src/components/structures/MatrixChat.js

```diff
@@ -1307,17 +1306,6 @@ export default React.createClass({
             return self._loggedInView.child.canResetTimelineInRoom(roomId);
         });
 
-        cli.on('sync.unexpectedError', function(err) {
```

Comment:
does it make sense to keep emitting this event in the sync as nobody is listening to it?

Reply:
It's in a generic place and survived long enough to land in a release. It'd be harder to deprecate than it is to just leave it for if anyone happens to be using it